app/controllers/shared/api.php

```diff
@@ -191,6 +193,17 @@
             ->setParam('project.{scope}.network.inbound', 0)
             ->setParam('project.{scope}.network.outbound', 0);
 
+        $smtp = $project->getAttribute('smtp', []);
```

Comment:
Do we need to reset this across requests? Do we need to reset the env setting so project Y doesn't use project X SMTP?

Reply:
isn't `$project` already reset ? If not how do you think we can do that ?